configuration/appsettings.rst

```diff
@@ -442,6 +442,18 @@ See :ref:`feature_multiversion`.
 
 .. _fhir_capabilities:
 
+Patient Everything Operation
+-----------------
+::
+
+  "PatientEverythingOperation": {
+    "AdditionalResources": [ "Organization", "Location", "Substance", "Medication", "Device" ] // included referenced resources, additional to the Patient compartment resources
+  },
+
+The patient everything operation returns all resources linked to a patient that are listed in the Compartment Patient. This section allows you to define additional resources that will be included in the resulting searchset bundle. 
```

Comment:
Can you put a reference here to https://docs.fire.ly/projects/Firely-Server/en/latest/features/patienteverything.html#configuration?

I would extend the $everything section with the comment below

Reply:
Done!